### PR TITLE
Support use_cached_catalog, pxp ping_interval, and use pxp-module-puppet from package

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
@doug-rosser

File: manifests/users.pp [hunk 1]
```diff
@@ -6,14 +6,32 @@
...
+  $run_interval   = $clamps::agent::run_interval,
   $splay          = false,
   $splaylimit     = undef,
 ) {
 
-  $user_cron_minute = clamps_user_number($user) % 30
-
-  $cron_1 = $user_cron_minute
-  $cron_2 = $user_cron_minute + 30
+  if $run_interval == 30 {
```
Comment (doug-rosser) [inline manifests/users.pp]: crond - why you so old-timey?

Comment (MikaelSmith) [inline manifests/users.pp]: Yeah. I want to make something better for turning an interval into a cron config. This was my hack for now.

**Comment by doug-rosser** [comment]
👍  is this ready for merge?

**Comment by MikaelSmith** [comment]
I think so. I'm using it and seems stable.

**Pull request merged**